templates/instance/admin.rs.html

```diff
@@ -17,7 +17,11 @@ <h1>@i18n!(ctx.1, "Administration")</h1>
   ])
 
   <form method="post" action="@uri!(instance::update_settings)">
-    @input!(ctx.1, name (text), "Name", form, errors.clone(), "props")
```

Comment:
what was `props`, why is it gone?

Reply:
I don't know to be honest, but it clearly was here by mistake (this argument was for additional HTML properties).